spec/benchmarks/benchmark_helper.rb

```diff
@@ -7,7 +7,7 @@ def iterate(&block)
     start = Time.now
     count = 0
     while Time.now - start <= 1.second do
-      block.call
+      yield
```

Comment:
Would you remove the `(&block)` argument in line 6? `yield` would not need that argument.